default.nix

```diff
@@ -1,32 +1,11 @@
-# This is the main nix file for termonad.  It will just build the termonad binary.
-# It can be built with the command `nix-build` in the main top directory.
+# This is the main nix file for termonad.  It will build and wrap the termonad
+# binary. It can be built with the command `nix-build` in the main top
+# directory.
 #
-# The termonad binary will be created at `result/bin/termonad`.
+# The termonad executable will be created at `result/bin/termonad`.
 
 { compiler ? "ghc843" }:
 
-let
-  nixpkgs = import ./.nix-helpers/nixpkgs.nix;
+(import <nixpkgs> {}).callPackage
```

Comment:
Is there a reason why you're using `callPackage` from the user's system `nixpkgs`, instead of the `nixpkgs` being used internally by termonad?

Reply:
No particularly good reason. At this point the arguments sensitive to the nixpkgs version have already been passed in, so I just used the `callPackage` that I was previously using in the overlay I had calling `termonad-with-packages.nix`.